roles/wazo_vars/defaults/main.yml

```diff
@@ -22,3 +35,5 @@ tenant_name: my-company
 api_client_name: api-client
 api_client_password: api-password
 
+wazo_debian_repo_upgrade: main
+wazo_distribution_upgrade: wazo-dev-stretch
```

Comment:
The first block of variables is a bit heteroclit. Could you split the first block?

Reply:
done